resources/views/app/master.blade.php

```diff
@@ -44,11 +45,16 @@
         <link href = "{!! load_asset('/library/sweetalert/sweetalert.css') !!}" rel="stylesheet"  />
         <script type="text/javascript" src = "{!! load_asset('/library/sweetalert/sweetalert.min.js') !!}"></script>
 
+        <script type="text/javascript" src = "{!! asset('/js/LoginAndSignup.js') !!}"></script>
+        <script type="text/javascript" src = "{!! asset('/js/PasswordReset.js') !!}"></script>
+        <script type="text/javascript" src = "{!! asset('/js/NewPassword.js') !!}"></script>
```

Comment:
**Code Functionality - Nitpick**

Please replace asset with load_asset to ensure it sets HTTPS on production environment for referencing the scripts.